packages/datadog-instrumentations/src/express.js

```diff
@@ -106,7 +106,17 @@ const wrapProcessParamsMethod = (requestPositionInArguments) => {
   return (original) => {
```

Comment:
Can we make wrapProcessParamsMethod a non arrow function, and add names to the functions under that ? it's easier to reason about with actual function names.
Usually it goes like 
`wrapProcessParams()`
-> `wrappedProcessParams()`
-> -> `processParamsCallback()`
or something like that

Reply:
✅